openfl-workspace/flower-app-pytorch/README.md

```diff
@@ -0,0 +1,291 @@
+# Open(FL)ower
```

Comment:
Nice

Reply:
@psfoley, this is Flower Summit slide material haha 😆